frontend/src/features/admin-form/settings/components/AuthSettingsSection/FormSingleSubmissionToggle.tsx

```diff
@@ -0,0 +1,63 @@
+import { useCallback } from 'react'
+import { Link as ReactLink } from 'react-router-dom'
+import { Text } from '@chakra-ui/react'
+
+import { PaymentChannel } from '~shared/types'
+import { FormResponseMode, FormSettings } from '~shared/types/form'
+
+import InlineMessage from '~components/InlineMessage'
+import Link from '~components/Link'
+import Toggle from '~components/Toggle'
+
+import { useMutateFormSettings } from '../../mutations'
+
+interface FormSingleSubmissionPerNricToggleProps {
+  settings: FormSettings
+  isDisabled: boolean
+}
+
+export const FormSingleSubmissionToggle = ({
+  settings,
+  isDisabled,
+}: FormSingleSubmissionPerNricToggleProps): JSX.Element => {
+  const { mutateIsSingleSubmission } = useMutateFormSettings()
+  const isSingleSubmission = !!settings?.isSingleSubmission
+  const isLoading = !settings || mutateIsSingleSubmission.isLoading
+
+  const isPaymentsEnabled =
+    settings &&
+    settings.responseMode === FormResponseMode.Encrypt &&
+    ((settings?.payments_channel?.channel &&
+      settings.payments_channel.channel !== PaymentChannel.Unconnected) ||
+      settings?.payments_field?.enabled)
```

Comment:
Looks like there are quite a few unnecessary optional chaining used.

```suggestion
    (settings.payments_channel.channel !== PaymentChannel.Unconnected ||
      settings.payments_field.enabled)
```

Reply:
removed!